src/capi_frontend/capi.cpp

```diff
@@ -1136,6 +1311,14 @@ DLL_PUBLIC void OVMS_ServableMetadataDelete(OVMS_ServableMetadata* metadata) {
     delete reinterpret_cast<ovms::ServableMetadata*>(metadata);
 }
 
+OVMS_Status* OVMS_ServerInitilizeGlobalVADisplay(void* vaDisplay) {
```

Comment:
Is it really initialization? I think "SetGlobalVADisplay", or maybe "Reset" if it is possible to reset